### ISSUE TITLE
Resizing doesn't seem to re-flow content

### ISSUE THREAD

**Issue by ijstokes** [Anaconda-Platform/nbpresent#68]
See this screencast:

http://screencast.com/t/7ltl4z9pK

Resizing the window doesn't re-flow the content, it just expands outside of the view port.

Going forward a slide and then back (at the "increased font size") causes a re-flow so the content looks fine.

It would be nice if the re-flow could happen immediately.

**Comment by bollwyvl** [comment]
Yeah, we can listen for resize. It will have to be debounced, as our
layout/theme mechanism is not cheap... So it will be perhaps 200ms after
you stop resizing. Indeed, we might want the resize to update a modeled
version of screen measurements, and just have the rerender be a watcher on
that.
